docs/using/selecting.md

```diff
@@ -142,8 +142,10 @@ You must refer to git-SHA image tags when stability and reproducibility are impo
 
 The core stacks are just a tiny sample of what's possible when combining Jupyter with other technologies. We encourage members of the Jupyter community to create their own stacks based on the core images and link them below.
 
-*This list only have 1 example. You can be the next!*
+*This list only have 2 examples. You can be the next!*
```

Comment:
```suggestion
*This list only has 2 examples. You can be the next!*
```

Reply:
Nice, I never used the suggestion more... interesting!